src/app/modules/recipe-edit/services/recipe-edit.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, throwError } from 'rxjs';
+import {catchError} from 'rxjs/operators';
+import {Recipe} from "../../recipe-create/models/recipe-create.model";
+
+@Injectable({
```

Comment:
Remove `providedIn: 'root'
```suggestion
@Injectable()
```
and add `providers: [RecipeEditService]` to `RecipeCreateModule`